### DISCUSSION TITLE
Ultimaker plugin

### DISCUSSION THREAD

**Discussion by idowantosell** [OctoFarm/OctoFarm#1295] [Feature Request]
Hi.
It would be nice to have a plugin that can connect and control a Ultimaker printer (i.e. Ultimaker 2) from OctoFarm (even if it does not require a raspberry pi / octoprint install)

API is available and camera would be able to be access too

Thanks

**Comment by NotExpectedYet** [comment]
This falls under the same requests for supporting other printing interfaces. Currently OctoPrint support is too coupled for this to be even acheivable in a decent amount of time sadly. 

I'm happy for a PR for the support but until there's been some major refactoring of the code then it will not be possible. This is slated for V2.0 work but is not going to be anytime soon.

**Comment by NotExpectedYet** [comment]
Please see #1298
